src/colors.scss

```diff
@@ -0,0 +1,335 @@
+            --ds-auro-button-text-color: var(--ds-color-text-ui-disabled-inverse, #{$ds-color-text-ui-disabled-inverse});
+          }
+        }
+      }
+    }
+
+    &[variant='tertiary'] {
+      --ds-auro-button-container-color: var(--ds-color-container-ui-tertiary-default-inverse, #{$ds-color-container-ui-tertiary-default-inverse});
+      --ds-auro-button-container-image: var(--ds-color-container-ui-tertiary-default-inverse, #{$ds-color-container-ui-tertiary-default-inverse});
+      --ds-auro-button-border-color: transparent;
+      --ds-auro-button-text-color: var(--ds-color-text-ui-default-inverse, #{$ds-color-text-ui-default-inverse});
+
+      @media (hover: hover) {
+        &:active,
+        &:hover {        
+          --ds-auro-button-container-color: var(--ds-color-container-ui-tertiary-hover-inverse, #{$ds-color-container-ui-tertiary-hover-inverse});
+          --ds-auro-button-container-image: var(--ds-color-container-ui-tertiary-hover-inverse, #{$ds-color-container-ui-tertiary-hover-inverse});
+          --ds-auro-button-border-color: transparent;
+          --ds-auro-button-text-color: var(--ds-color-text-ui-hover-inverse, #{$ds-color-text-ui-hover-inverse});
+        }
+      }
+
+      &:focus-visible {
+        --ds-auro-button-border-color: var(--ds-color-border-ui-default-inverse, #{$ds-color-border-ui-default-inverse});
+        --ds-auro-button-border-inset-color: var(--ds-color-border-ui-default-inverse, #{$ds-color-border-ui-default-inverse});
+      }
+
+      &:active {
+        --ds-auro-button-container-color: var(--ds-color-container-ui-tertiary-active-inverse, #{$ds-color-container-ui-tertiary-active-inverse});
+        --ds-auro-button-container-image: var(--ds-color-container-ui-tertiary-active-inverse, #{$ds-color-container-ui-tertiary-active-inverse});
+        --ds-auro-button-border-color: transparent;
+        --ds-auro-button-text-color: var(--ds-color-text-ui-hover-inverse, #{$ds-color-text-ui-hover-inverse});
+      }
+
+      &:disabled {
+        --ds-auro-button-container-color: var(--ds-color-container-ui-tertiary-disabled-inverse, #{$ds-color-container-ui-tertiary-disabled-inverse});
+        --ds-auro-button-container-image: var(--ds-color-container-ui-tertiary-disabled-inverse, #{$ds-color-container-ui-tertiary-disabled-inverse});
+        --ds-auro-button-border-color: transparent;
+        --ds-auro-button-text-color: var(--ds-color-text-ui-disabled-inverse, #{$ds-color-text-ui-disabled-inverse});
+
+        @media (hover: hover) {
+          &:hover {
+            --ds-auro-button-container-color: var(--ds-color-container-ui-tertiary-disabled-inverse, #{$ds-color-container-ui-tertiary-disabled-inverse});
+            --ds-auro-button-container-image: var(--ds-color-container-ui-tertiary-disabled-inverse, #{$ds-color-container-ui-tertiary-disabled-inverse});
+            --ds-auro-button-border-color: transparent;
```

Comment:
Sort of getting lost in the code, but is there any reason you are setting the `border` color to `transparant` everywhere, rather than setting it to the same color as the `container` color? Remember the conversation we had about keeping the box model.

Reply:
Must be transparent with the opacity colors due to the border rendering on top of the background. Not using transparent the layering causes the border to appear with a higher opacity level.